netlify.toml

```diff
@@ -1,13 +1,13 @@
 [build]
 publish = "public"
-command = "make production-build"
+command = "git submodule update --init --recursive --depth 1 && make production-build"
 
 [build.environment]
 HUGO_VERSION = "0.79.0"
 NODE_VERSION = "12"
 
 [context.deploy-preview]
-command = "make preview-build"
+command = "git submodule update --init --recursive --depth 1 && make preview-build"
 
 [context.branch-deploy]
-command = "make preview-build"
+command = "git submodule update --init --recursive --depth 1 && make preview-build"
```

Comment:
We don't want to fetch the latest `docsy` repo version (which is what the `git submodule update ...` command does AFAIK), we want to stick to the submodule version that is associated with this repo via `.gitmodules`. Otherwise a build on Netlify might (unknown to us) be different from local builds.

AFAIK, it shouldn't be necessary to add this extra `git` command (unless the `docsy` repo itself has submodules).

Reply:
Was the build breaking w/o the addition of the `git submodule` commands?

Reply:
(We don't need to address this before this is merged, only before the final PR.)

Reply:
Yeah, its the `docsy` submodules. I wonder if there's a better solution (Netlify config parameter), or do all docsy-based projects to this? (No rush to answer; we'll address this after the merge.)

Reply:
Seems like a good (recommended) solution. Marking as resolved.

Reply:
I was just testing this out [removing the change] and it seems to still work when I run `netlify build` and `netlify deploy` to our staging. Previously I'd added something along these lines in order for the submodule to install [Docsy contains submodules for `Bootstrap` and `Font-Awesome`]. I've reverted this and will push my recent commits, observing the output directly on Netlify.

Reply:
Thanks for trying this. It might be working now because of the Netlify cache. I'm going to test that hypothesis by initiating a cache-clear-and-rebuild comment, and we can see how it goes.

Reply:
@aidanranney - The build is failing now that the cache has been cleared, so we'll need those commands after all 🤷‍♂️.

```console
8:55:05 PM: ────────────────────────────────────────────────────────────────
8:55:05 PM:   1. build.command from netlify.toml                            
8:55:05 PM: ────────────────────────────────────────────────────────────────
8:55:05 PM: ​
8:55:05 PM: $ make preview-build
8:55:05 PM: rm -rf public/* resources
hugo --baseURL https://deploy-preview-561--grpc-io.netlify.app --minify
8:55:05 PM: Start building sites …
8:55:07 PM: Total in 2678 ms
8:55:07 PM: Error: Error building site: TOCSS: failed to transform "scss/main.scss" (text/x-scss): SCSS processing failed: file "stdin", line 8, col 1: File to import not found or unreadable: ../vendor/bootstrap/scss/bootstrap.
8:55:07 PM: Makefile:29: recipe for target 'preview-build' failed
8:55:07 PM: make: *** [preview-build] Error 255
```

Reply:
@chalin Thanks for narrowing that down - I'll drop the revert commit momentarily.